reinforcement_learning/rlclientlib/logger/event_hub_logger.h

```diff
@@ -9,68 +10,72 @@
 #include "api_status.h"
 #include "../error_callback_fn.h"
 #include "utility/watchdog.h"
+#include "ranking_response.h"
+#include "ranking_event.h"
 
 namespace reinforcement_learning {
   // This class wraps logging event to event_hub in a generic way that live_model can consume.
-  class event_hub_logger : public i_logger {

```

Comment:
>event_hub_logger [](start = 8, length = 16)

Placeholder:  Wait for changes discussed offline.  (name changes and moving periodic_proc to event_logger.